File: 1603-running-sum-of-1d-array/running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        pfx = [0] * len(nums)
        pfx[0] = nums[0]
        for i in range(1,len(nums)):
            pfx[i] = pfx[i-1] + nums[i]
        return pfx
        

File: 1603-running-sum-of-1d-array/running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) 
    {
        int n = nums.length;
        int[] pfx= new int[n];

        pfx[0] = nums[0];

        for(int i=1; i < n ; i++)
        {
            pfx[i] = pfx[i-1] + nums[i];
        }

        return pfx;
        
    }
}